<p>
    Boolean variable should not add prefix is
</p>
<h2>Noncompliant Code Example</h2>
<pre>
    public class DemoDO{
        Boolean isSuccess;
        Boolean isDelete;
    }
</pre>
<h2>Compliant Solution</h2>
<pre>
    public class DemoDO{
        Boolean success;
        Boolean delete;
    }
</pre>