<p>
    Package should be named in lowercase characters
</p>
<h2>Noncompliant Code Example</h2>
<pre>
    package org.sonar.java.rule.checks.Namerules;

    public class PackageNamingEx {

    }
</pre>
<h2>Compliant Solution</h2>
<pre>
    package org.sonar.java.rule.checks.namerules;

    public class PackageNamingEx {

    }
</pre>